subsys/bluetooth/mesh/blob_cli.c

```diff
@@ -1521,9 +1519,9 @@ int bt_mesh_blob_cli_send(struct bt_mesh_blob_cli *cli,
 		return -EBUSY;
 	}
 
-	if (!(xfer->mode & BT_MESH_BLOB_XFER_MODE_ALL) ||
-	    xfer->block_size_log < 0x06 || xfer->block_size_log > 0x20 ||
-	    xfer->chunk_size < 8 || xfer->chunk_size > CHUNK_SIZE_MAX) {
+	if (!(xfer->mode & BT_MESH_BLOB_XFER_MODE_ALL) || xfer->block_size_log < 0x06 ||
+	    xfer->block_size_log > 0x20 || xfer->chunk_size < 8 ||
+	    xfer->chunk_size > CONFIG_BT_MESH_TX_BLOB_CHUNK_SIZE) {
```

Comment:
I think this should be now:
```suggestion
	    xfer->chunk_size > cli->caps.max_cunk_size) {
```

Reply:
This seems sensible, but at least for the tests, the caps aren't always set, as they are only filled in when `bt_mesh_blob_cli_caps_get` or when the BLOB info status is requested. I am not sure if perhaps `bt_mesh_blob_cli_caps_get` should be invoked during blob client init? Or I should revert this suggested change.